pygmt/tests/test_image.py

```diff
@@ -1,14 +1,11 @@
 """
 Test Figure.image.
 """
-import os
 import sys
 
 import pytest
 from pygmt import Figure
 
-TEST_IMG = os.path.join(os.path.dirname(__file__), "baseline", "test_logo.png")
-
 
 @pytest.mark.skipif(sys.platform == "win32", reason="crashes on Windows")
```

Comment:
Gonna try removing this line and see if it still crashes. Xref https://github.com/GenericMappingTools/pygmt/pull/518#issuecomment-657896532.

Reply:
Oo, `test_image` no longer crashes on Windows with GMT 6.3 (see https://github.com/GenericMappingTools/pygmt/actions/runs/7334833328/job/19971998592?pr=2922#step:7:508) and GMT 6.4 (see https://github.com/GenericMappingTools/pygmt/actions/runs/7334833574/job/19971999909?pr=2922#step:8:473) :tada: